### ISSUE TITLE
Issue with vorta tray icon on Ubuntu 18.04

### ISSUE THREAD

**Issue by homebase3** [borgbase/vorta#654]
**Describe the bug**

**To Reproduce**
1. Install Vorta on Ubuntu 18.04 with Pantheon desktop environment (or Elementary OS 5.1 with [tray icons enabled](https://eos-techs.com/2020/05/19/howto-enable-system-tray-on-elementary-os/))
2. Open Vorta
3. Abnormal tray icon appears (white box with grey circle and line strike-through)
![Screenshot from 2020-09-17 09 44 17](https://user-images.githubusercontent.com/65870950/93479508-63d1d680-f8ca-11ea-891e-9b4306996cc2.png)

**Desktop (please complete the following information):**
- OS: Ubuntu 18.04 (Pantheon desktop version)
- Vorta version: 0.7.1
- Borg version: 1.1.13

**Additional context**
Logs likely not relevant here, but attached just in case:
[vorta.log](https://github.com/borgbase/vorta/files/5239487/vorta.log)

**Comment by samuel-w** [comment]
Are you using flatpak, because it could be related to #318

**Comment by homebase3** [comment]
Yes I am. I was able to install vorta via pip3 but now can't figure out how to launch the application (it doesn't show up in my application menu).I tried to follow the instructions outlined [here](https://vorta.borgbase.com/install/linux/), but after building a vorta.desktop file in the directory specified with the text written per the [template](https://github.com/borgbase/vorta/blob/master/src/vorta/assets/metadata/com.borgbase.Vorta.desktop), it still didn't work.

Any idea why? Should I be changing the exec path? If so, what should I change it to?

**Comment by samuel-w** [comment]
You need to add the bin to you path, add the following to your ~/.profile file and restart
```
if [ -d "$HOME/.local/bin" ] ; then
    PATH="$PATH:$HOME/.local/bin"
fi
```

**Comment by homebase3** [comment]
I tried this procedure and unfortunately it didn't resolve the issue. If it's helpful, from what I can tell, Vorta's program files are stored on my system (post-installation via pip3) at the following path: 
```
~/.var/app/com.borgbase.Vorta
```

**Comment by samuel-w** [comment]
Is there anything prevent you from upgrading to 20.04? Because I have a Vorta [PPA](https://launchpad.net/~samuel-w1/+archive/ubuntu/vorta) that works for 20.04.

**Comment by samuel-w** [comment]
Can you run it through `~/.local/bin/vorta`?

**Comment by samuel-w** [comment]
Fixed by #676
